src/controller/launcher/webots_controller.c

```diff
@@ -223,7 +220,7 @@ static bool get_matlab_path() {
   const size_t matlab_path_size = snprintf(NULL, 0, "%s%s%s", matlab_directory, latest_version, matlab_exec_suffix) + 1;
   matlab_path = malloc(matlab_path_size);
   sprintf(matlab_path, "%s%s%s", matlab_directory, latest_version, matlab_exec_suffix);
-  printf("Using matlab path %s\n",matlab_path);
+  printf("MATLAB instance found at: %s\n", matlab_path);
```

Comment:
This

Reply:
This is a good point. I agree! However, 
- MATLAB should be capitalized in order to be consistent. 
- The colon should be left in order to be consistent with
`Using prototype file: C:\msys64\tmp\protofile_matlab_2021a_webots_R2023b_revision.m`
- The space between args (col 36) will cause an error in clang-format

Reply:
`printf("Using the latest available MATLAB instance: %s\n", matlab_path);`